features/support/hooks.js

```diff
@@ -37,33 +36,31 @@ module.exports = function () {
     // application server mock
     this.app = makeExpressApp(this)
     this._server = http.createServer(this.app)
-    var primusOpts = {
+    const primusOpts = {
```

Comment:
Perhaps this is an older style of development that I've gained when creating variables IN_ALL_CAPS. things IN_ALL_CAPS should probably be raised to the top of the file (at least, historically in my usage). This turns into an exercise in personal preference. I'll leave it be for now and we can go with `const` here.